src/main/resources/org/rust/ide/liveTemplates/output.xml

```diff
@@ -31,6 +31,24 @@
         </context>
     </template>
 
+    <template name="pdv" description="debug println! value" toReformat="true" toShortenFQNames="true"
+              value="println!(&quot;$EXPRESSION_COPY$ = {:?}&quot;, $EXPRESSION$);">
```

Comment:
It's not yet in nightly and it didn't get into the current alpha as well. 

As for the community feedback, I think we have a general problem that we receive little feedback :( And I suspect this is in part because not everyone understands what features are already available, and what should be possible. 


So yes, it would be nice to collect info about this particular feature (and to learn what part of users know about this snippets!)